### PR TITLE
Fix styles

### PR DESCRIPTION
**Pull request by thinca**
These are reported by vint 0.3.4.

### CODE REVIEW

File: test/Data/String.vim [hunk 1]
```diff
@@ -274,7 +274,6 @@ function! s:suite.dstring()
   " wraps the result string not with single-quotes but with double-quotes.
   call s:assert.equals( s:String.dstring(123), '"123"')
   call s:assert.equals( s:String.dstring('abc'), '"abc"')
-  call s:assert.equals( s:String.dstring("abc"), '"abc"')
```
Comment (thinca) [inline test/Data/String.vim]: I removed this because this is same as above.

File: autoload/vital/__latest__/Data/String.vim [hunk 2]
```diff
@@ -371,7 +371,7 @@ function! s:split_by_displaywidth(expr, width, float, is_wrap) abort
 
   let text = ''
   while cs_index < len(cs)
-    if cs[cs_index] is "\n"
+    if cs[cs_index] is# "\n"
```
Comment (tyru) [inline autoload/vital/__latest__/Data/String.vim]: 文字列ってisで比較するのありなんだ…(あとで確認)

Comment (tyru) [inline autoload/vital/__latest__/Data/String.vim]: ありなんだ…(確認ed)

**Comment by tyru** [comment]
LGTM

**Pull request merged**